packages/data-service/src/instance-detail-helper.spec.ts

```diff
@@ -89,6 +89,10 @@ describe('instance-detail-helper', function () {
             null
           );
         });
+
+        it('should not be identified as atlas', function () {
+          expect(instanceDetails).to.have.property('isAtlas', false);
+        });
```

Comment:
Maybe it makes sense to add a few test cases with the mocked client below to check the outcome where command succeeds?